blog/models.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Post(models.Model):
+    owner = models.ForeignKey("User", on_delete=models.CASCADE)
+    title = models.CharField(max_length=200)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.title
+
+    @property
+    def comment_count(self):
+        return self.commentary_set.count()
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
Similar to the Post model, the ForeignKey reference to 'User' should use the AUTH_USER_MODEL setting instead of a hardcoded string. This ensures compatibility with custom user models.